consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundState.java

```diff
@@ -80,15 +80,13 @@ public boolean setProposedBlock(final Proposal msg) {
   public void addPrepareMessage(final Prepare msg) {
     if (!proposalMessage.isPresent() || validator.validatePrepare(msg)) {
       prepareMessages.add(msg);
-      LOG.debug("Round state added prepare message prepare={}", msg);
```

Comment:
nit: not even trace? ... maybe not ... (just want to confirm!)

Reply:
done. made it trace